doc/papers/joss/paper.md

```diff
@@ -50,7 +50,7 @@ affiliations:
    index: 2
  - name: University of Tennessee
    index: 3
- - name: Brex
+ - name: Occalytics LLC
```

Comment:
@dmed256 I had found a different affiliation for you on the internet, but Tzanio suggested Occalytics LLC for the MS-34 report. Could you please confirm or change that at your convenience? Thanks